measurements-db/src/main/java/com/expidevapps/android/measurements/db/GmaDatabase.java

```diff
@@ -70,9 +70,13 @@
      * 51: 2015-09-30
      * 52: 2015-10-06
      * 53: 2015-10-14
+     * v1.0.0
+     * 54: 2015-10-20
+     * 55: 2015-10-27
+     * 56: 2015-11-02
      */
     private static final String DATABASE_NAME = "gcm_data.db";
-    private static final int DATABASE_VERSION = 53;
+    private static final int DATABASE_VERSION = 56;
```

Comment:
I had been sending debug versions to Mike Thacker as I went, plus it makes local installs easier because I don't need to wipe the db to work with it.